Makefile

```diff
@@ -0,0 +1,33 @@
+.DEFAULT_GOAL = help
+
+NAME = storybook-dev
+EXISTING_CONTAINERS = $$(docker ps --all --quiet --filter "name=${NAME}")
+
+help: ## Display this help text
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+install: ## Install dependencies locally
+	npm install
+
+build: ## Build the container
+	docker build --target dev --tag libero/storybook:latest .
```

Comment:
Maybe `dev` rather than `latest`?

Reply:
Looking ahead to https://github.com/libero/storybook/pull/4, might be useful to use an environment variable to say `dev` or `prod` for _all_ commands.